backend/analytics_server/mhq/api/pull_requests.py

```diff
@@ -167,3 +167,22 @@ def get_team_lead_time_trends(
         week.isoformat(): adapt_lead_time_metrics(average_lead_time_metrics)
         for week, average_lead_time_metrics in weekly_lead_time_metrics_avg_map.items()
     }
+
+
+@app.route("/teams/<team_id>/prs/merged_without_review", methods={"GET"})
+@queryschema(
+    Schema(
+        {
+            Required("from_time"): All(str, Coerce(datetime.fromisoformat)),
+            Required("to_time"): All(str, Coerce(datetime.fromisoformat)),
+        }
+    ),
```

Comment:
@RISHIKESHk07 we additionally have a PR filter that we use in query params which is missing here. PR filter helps us add more querying from branch, author etc at a api level by just adding handler for the query.

Reply:
@samad-yar-khan  so created_at & updated_at should fix this , please correct me if i am wrong

Reply:
yes, PR filter is applied at repo, so you will have to pass it down funcs as argument